02-Custom-Login/src/views/Main/Login/Login.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes as T } from 'react'
+import ReactDOM from 'react-dom'
+import {Form, FormGroup, FormControl, ControlLabel, Button, ButtonToolbar} from 'react-bootstrap'
+import AuthService from 'utils/AuthService'
+import styles from './styles.module.css'
+
+export class Login extends React.Component {
+  static contextTypes = {
+    router: T.object
+  }
+
+  static propTypes = {
+    location: T.object,
+    auth: T.instanceOf(AuthService)
+  }
+
+  handleSubmit(e){
+    e.preventDefault()
+    this.props.auth.login({
+      connection: 'Username-Password-Authentication',
+      responseType: 'token',
+      email: ReactDOM.findDOMNode(this.refs.email).value,
+      password: ReactDOM.findDOMNode(this.refs.password).value
```

Comment:
@flaviogranero perahps you can extract this to a method as it's the same as the params sent to signUp.